dlrover/python/unified/common/workload_base.py

```diff
@@ -92,8 +92,11 @@ def self_check(self):
         logger.info(f"[{self.node_info.name}] Running self check.")
```

Comment:
'workload_base.py' rename to 'actor_base.py'

Reply:
Keep it consistent with `workload_desc` temporarily, and clean it up together later.